datapackage_pipelines_mojp/common/processors/sync.py

```diff
@@ -15,7 +15,8 @@
                                     {'name': 'id', 'type': 'string'},
                                     {"name": "version", "type": "string",
                                      "description": "source dependant field, used by sync process to detect document updates"},
-                                    {"name": "collection", "type": "string", "description": COLLECTION_FIELD_DESCRIPTION},
+                                    {"name": "collection", "type": "string",
+                                        "description": COLLECTION_FIELD_DESCRIPTION},
                                     {"name": "source_doc", "type": "object"},
                                     {"name": "title", "type": "object",
                                      "description": "languages other then he/en, will be flattened on elasticsearch to content_html_LANG"},
```

Comment:
oh, actually, I see that the lower-case data is only in elasticsearch - in that case, no need to change the schema